extension/src/extension.ts

```diff
@@ -22,7 +22,11 @@ function createParser(): Parser {
         throw new Error("Process runtime limit not configured");
       }
       return {
-        executable: ifmConfig.get("executablePath") || "ifm",
+        executable:
+          /* eslint-disable-next-line
+             @typescript-eslint/prefer-nullish-coalescing
+          -- falsy is what we want here because of the empty string */
```

Comment:
Note to self: I’m going to need those three lines often enough that this might warrant a snippet.